src/generic/richtooltipg.cpp

```diff
@@ -245,6 +245,15 @@ class wxRichToolTipPopup :
     void DoShow()
     {
         Popup();
+
+        // Call Popup function in case of an already showing rich tooltip 
+        // lead to closing the previous tooltip because of losing focus
+        // while showing the current one. It's lead to losing focus from
```

Comment:
Could you please explain why does the current popup lose focus when the old popup is dismissed? This seems like the real problem to me and while we can work around it by resetting the focus here, it would be even nicer if we could avoid losing focus in the first place.